### PR TITLE
Iteration 3 mb

### PR DESCRIPTION
**Pull request by Maxwell-Baird**
Created four methods called lowest scoring visitor, lowest scoring home team, highest scoring visitor, highest score home team.
These four methods call upon scoring which sorts and creates a hash of average goals for either home or away, scoring then calls upon low or high that returns either the highest score or the lowest score from the average goals.
Also created a method that takes a team id and returns team name

### CODE REVIEW

File: lib/game_teams_stats.rb [hunk 1]
```diff
@@ -7,6 +7,62 @@ class GameTeamStats
...
+    scoring('home','low')
+  end
+
+  def highest_scoring_visitor
+    scoring('away','win')
+  end
+
+  def highest_scoring_home_team
+    scoring('home','win')
+  end
```
Comment (alex-latham) [inline lib/game_teams_stats.rb:27]: This astounds me. Great job with using a single helper method for simplify each of these stat methods.

File: lib/team_stats.rb [hunk 2]
```diff
@@ -9,4 +9,9 @@ def initialize(file_path, object)
     @teams = csv_data('./data/teams.csv', Team)
   end
 
+  def find_name(id)
```
Comment (whitneykidd) [inline lib/team_stats.rb:12]: This method is so helpful! 👍

File: lib/game_teams_stats.rb [hunk 3]
```diff
@@ -7,6 +7,62 @@ class GameTeamStats
 
   def initialize(file_path, object)
     @game_teams = csv_data(file_path, object)
+    @team_stats = TeamStats.new("./data/teams.csv", Team)
```
Comment (tylertomlinson) [inline lib/game_teams_stats.rb:10]: I wonder how this will affect our spec harness. Anxious to see.

File: lib/game_teams_stats.rb [hunk 4]
```diff
@@ -7,6 +7,62 @@ class GameTeamStats
...
   end
 
+  def lowest_scoring_visitor
+    scoring('away','low')
+  end
+
+  def lowest_scoring_home_team
```
Comment (tylertomlinson) [inline lib/game_teams_stats.rb]: MR. CLEAN doing his magic

File: lib/game_teams_stats.rb (hunk 1 above)
Comment (tylertomlinson) [inline lib/game_teams_stats.rb:27]: MR. CLEAN doing his magic

File: lib/team_stats.rb (hunk 2 above)
Comment (tylertomlinson) [reply lib/team_stats.rb:12]: ^^^^

**Pull request merged**